src/ThirdParty/Gitlab/Adapter/GitLabIssueTracker.php

```diff
@@ -190,7 +196,18 @@ public function getComments($id)
             $this->client->api('issues')->show($this->getCurrentProject()->id, $id)
         );
 
-        return $issue->showComments();
+        $comments = [];
+        $issue->showComments();
```

Comment:
I know that this seems a bit weird but I do not use `showComments` return value directly.

There is a problem in the `php-gitlab-api` lib which does not allow to access the `id` of a comment in the properties.
So I need to do the following steps :

* Execute the request to retrieve comment list with `showComments`
* Access to the `HttpClient` last response with `getLastResponse` call
* Extract and retrieve all the requested details

I think I'll submit a PR on the `php-gitlab-api` project but before the changes will be accepted, we need to deal with that ugly stuff.